### DISCUSSION TITLE
transferring the project from one account to another account, and get Domain is pending verification warning

### DISCUSSION THREAD

**Discussion by freeflowthought** [vercel/community#5230] [Help]
### Summary

![vercel](https://github.com/vercel/community/assets/73615614/b627997c-c53e-4852-b769-8f54fe176c84)
that vercel account has deleted the project and the domain as well. (I actually managed to delete the project).
but I still get this error below

Another Vercel account is using this domain. Please set the following TXT record on _vercel.splatterprotocol.xyz to use www.splatterprotocol.xyz in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

Type
TXT
Name
_vercel
Value
vc-domain-verify=www.splatterprotocol.xyz,711854e736411baf440a

### Steps to Reproduce

1: set up a project and bind with a domain.
2: remove the domain with this project.
3:set up a new project and bind the same domain with this new project (the error should gone by now, but still there)
4: delete the entire old project, but new project still see the same problem

**Comment by freeflowthought** [comment]
I don't see any place in the setting where I can add type,name, and value

**Comment by amyegan** [comment]
Hi @freeflowthought! You can [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) for a third party domain by adding the requested TXT record. That will let you use the domain for your project. Another option if the team still exists, or if someone else from the team currently owns the domain, is to [transfer domain ownership](https://vercel.com/docs/projects/domains/transfer-your-domain#) to your account.

**Comment by freeflowthought** [comment]
![image](https://github.com/vercel/community/assets/73615614/34afd1d0-9fa3-4fe8-871e-72e6601858ec)
tried to add the txt record.  but don't see this place in anywhere.      currently the domain is already being transferred to my account.  it's godaddy domain.  but error still exists.

**Comment by freeflowthought** [comment]
I believe you have the some data to check.
in the back scene, I think you should see there is no link to the splatterprotocol.xyz anymore.
but I can't add it
